### ISSUE TITLE
Add OpenAPI to compendium

### ISSUE THREAD

**Issue by pirita** [higherkindness/compendium#30]
We should add OpenAPI to compendium

**Comment by pepegar** [comment]
openapi is already supported by skeuomorph, although not thoroughly tested. I think that the integration in compendium should be easy.

**Comment by Yawolf** [comment]
Is this the same as #99?

**Comment by JesusMtnez** [comment]
@mrtmmr what is the status on this one? Are all pieces ready to support `OpenAPI` ?

**Comment by mrtmmr** [comment]
No, it was on development but I had to stop to prioritize other issues. I guess this will disappear when we merge compendium plugin with mu plugin. If not I'll put an eye on it
